docs/blog/newsletter-april-2024.rst

```diff
@@ -0,0 +1,143 @@
+.. post:: April 04, 2024
+  :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – April 2024
+#########################################
+
+Hail to you, documentarians! I am writing with birds singing outside my window to celebrate the coming of spring (or so I tell myself). I hope you are able to find some sounds that bring you hope for the future.
+
+In the near future, the `Portland 2024 conference </conf/portland/2024/>`__ is just around the corner, starting on April 14th. I hope all of you who are able to attend come away with amazing ideas and connections. For the rest of us, Write the Docs Atlantic 2024 has now opened `ticket sales and grant applications </conf/atlantic/2024/news/tickets-grants/>`__. Plus the `Call for Proposals </conf/atlantic/2024/cfp/>`__ is still open until May 15th, so submit your proposals while you can.
+
+Speaking of conferences, we're furiously putting the finishing touches on the results of the 2023 Documentation Salary Survey so we can publish before the Portland conference starts. The best ever response for this survey has meant that there's been a mountain of really fascinating data to process, and we can't wait to share the insights with you! If you're going to be attending Writing Day in Portland this year, we're going to be running a session to improve the clarity, inclusivity, and relevance of the questions. 
```

Comment:
'best ever response' sounds  bit like one response, not the most responses, but might just be me

Reply:
I think it was my thought as well, but more so since I changed it a bit.
```suggestion
Speaking of conferences, we're furiously putting the finishing touches on the results of the 2023 Documentation Salary Survey so we can publish before the Portland conference starts. The most ever responses for this survey has meant that there's been a mountain of really fascinating data to process, and we can't wait to share the insights with you! If you're going to be attending Writing Day in Portland this year, we're going to be running a session to improve the clarity, inclusivity, and relevance of the questions. 
```